app/lib/account/models.dart

```diff
@@ -178,10 +178,6 @@ class Consent extends db.Model {
   /// The consent id.
   String get consentId => id as String;
 
-  /// The user that this consent is for.
```

Comment:
Also update the documentation comment for the `Content` class, it currently says `[User] (the parent entity).`